source/install-guide/installing-index.rst

```diff
@@ -31,7 +31,7 @@ Stop Elasticsearch using
 
 .. code-block:: shell
 
-    elasticsearch-7.6.2/bin/elasticsearch stop
+    elasticsearch-7.9.2/bin/elasticsearch stop
```

Comment:
does this represent the latest tested version? or should we rewrite this to indicate to use the latest.